renku/core/commands/config.py

```diff
@@ -48,6 +48,7 @@ def update_multiple_config():
         .command(_update_multiple_config)
         .require_migration()
         .with_commit(commit_if_empty=False, commit_only=CONFIG_LOCAL_PATH)
+        .with_database()
```

Comment:
```suggestion
        .command(_update_multiple_config)
        .with_database()
        .require_migration()
        .with_commit(commit_if_empty=False, commit_only=CONFIG_LOCAL_PATH)
```
I was using your PR while testing on #2234 , and I noticed updating the config from the service failed. Moving `.with_database()` before the other commands fixed it -- I copied it from other dataset commands that were succeeding

Reply:
Thanks for letting me know. Mohammad had a similar issue a while back. but the order shouldn't matter, it's quite weird.